__tests__/app.test.js

```diff
@@ -120,3 +120,45 @@ describe("GET /api/articles/:id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  it("200: responds with an array of comment objects containing the following properties; comment_id, votes, created_at, author, body & article_id", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        expect(body.comments.length).toBeGreaterThan(0);
+        expect(
+          body.comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number)
+            });
+            expect(comment.article_id).toBe(3);
+          })
+        );
+      });
+  });
+
+  it("400: returns an error when given an invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/invalid-id/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+
+  it("404: returns an error when given a valid article_id but there are no comments on the article", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No comments on that article");
+      });
+  });
+});
```

Comment:
This is a 200 not an error! If you want to render the comments on a facebook post you won't show an error, you will still just display the comments (0)